docs-src/product/release-stages/pre-release.md

```diff
@@ -0,0 +1,29 @@
+---
+id: pre-release
+title: Pre-release
+description: Learn more about the Pre-release stage
+sidebar_label: Pre-release
+tags:
+  - product release stages
+  - term
+---
+
+**Access:** Pre-release New Features are invite-only for Temporal Cloud customers, meaning Temporal will specifically select Temporal Cloud customers to be part of testing the Pre-release New Features.
+Temporal Cloud customers are invited to provide feedback via the Temporal engineering teams.
+
+Given the open source nature of Temporal, the Pre-release of New Features are available to all self-hosted Temporal customers, without the need for an invitation.
+While support from the Temporal team will not be provided for self-hosted Temporal customers, community support is available on Slack.
+
+**Classification:** Pre-release of New Features may not be fully mature and may have bugs.
+Customer acknowledges and agrees that the Pre-release New Features are provided on an “as-is” basis, and Pre-release
+New Features are provided without any indemnification, support, warranties, or representation of any kind.
+
+**Feedback:** Based on your participation in the Pre-release, feedback is requested but not required.
+Feedback is however highly encouraged and considered important to guiding Temporal’s New Feature development.
+Please share your experiences to influence the future direction of Temporal.
+
+**Availability:** Temporal may modify these New Features before release, or may even decide not to release them at all.
```

Comment:
Mild rephrasing suggestion: "Temporal may modify these New Features before release or may choose not to release them at all."